raiden-dapp/tests/e2e/specs/accept-disclaimer.spec.ts

```diff
@@ -1,11 +1,12 @@
-describe('disclaimer', () => {
-  it('accepts disclaimer and redirects to home route', () => {
-    cy.viewport('macbook-13');
-    cy.visit('/');
-    cy.contains('Disclaimer').should('be.visible');
-    cy.get('.disclaimer__accept-checkbox').click();
-    cy.get('.disclaimer__persist-checkbox').click();
-    cy.get('.disclaimer__accept-button').click();
-    cy.url().should('include', '/#/home');
-  });
+import { disclaimerRoute } from '../fixtures';
+
+it('accepts disclaimer and redirects to home route', () => {
+  disclaimerRoute();
```

Comment:
Could you add a verb to that function name? At the moment it is like "**What** disclaimer route?".
But very good approach to introduce this! :+1:

Reply:
Done!